### PR TITLE
Add Windows Imaging notes

### PR DESCRIPTION
**Pull request by brandondunne**

### CODE REVIEW

**Comment by brandondunne** [comment]
@mfeifer Please review

**Comment by brandondunne** [comment]
cc @gmcculloug @Fryguy

**Comment by mfeifer** [comment]
Looks good from formatting perspective :camel:

**Comment by brandondunne** [comment]
Bump :turtle:

File: provisioning/windows_imaging.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+1. Build Windows_PE environment and create ISO
+  1. Generate WinPE directory structure
+    * Open Start / Windows AIK / Deployment Tools Command Prompt
+    ```copycmd amd64 c:\winpe_amd64```
+  1. Copy additional tools and customize our working directory
+    * Always enter WinPE (don't prompt for key press)
+    ```del c:\winpe_amd64\ISO\boot\bootfix.bin```
+  1. Edit the Boot Image
+    * Mount the WIM file
+    ```dism /mount-wim /wimfile:c:\winpe_amd64\winpe.wim /index:1 /mountdir:c:\winpe_amd64\mount```
```
Comment (jrafanie) [inline provisioning/windows_imaging.md]: @brandondunne you might want to try multi-lining this snippet and the ones below with no indent as they are hard to read when they are wrapped.

Comment (jrafanie) [inline provisioning/windows_imaging.md]: Specifically, the bulleted line is immediately followed by the snippet.

See below:


screen shot 2015-02-13 at 10 56 03 am

**Comment by Fryguy** [comment]
@brandondunne Clean you clean up the part @jrafanie mentioned.  Then we can merge, I think.

**Comment by jrafanie** [comment]
@brandondunne ping, could you make the small change above?  Thanks!

**Comment by brandondunne** [comment]
@jrafanie Updated based on your comments

**Comment by jrafanie** [comment]
:+1:

**Pull request merged**